psydac/api/ast/linalg.py

```diff
@@ -434,6 +465,23 @@ def _compile_pyccel(self, mod, backend, verbose=False):
 
         return fmod
 
+#==============================================================================
+class InterfaceTransposeOperator(TransposeOperator):
```

Comment:
What does this class represent exactly? Please add a docstring

Reply:
this class accelerate the transpose code for a `StencilInterfaceMatrix`

Reply:
done in [96b6d86](https://github.com/pyccel/psydac/pull/160/commits/96b6d8668e7b352a3593f8938c3fc9dd1e155c48)